solutions/landing-zone/README.md

```diff
@@ -230,93 +236,169 @@ To deploy this Landing Zone you will first need to create a Bootstrap project wi
-                  auth: gcpserviceaccount
-                  gcpServiceAccountEmail: config-sync-sa@PROJECT_ID.iam.gserviceaccount.com
-              ```
-
-          4. Deploy the Config Sync Manifest
-              ```
-              kubectl apply -f root-sync.yaml
-              kubectl wait --for condition=established --timeout=10s crd/rootsyncs.configsync.gke.io
-              ```
-
-          5. Push Configs to Git
-              ```
-              gcloud source repos clone my-lz-repo
-              cd my-lz-repo
-              ```
-
-              Modify the `setters.yaml` file and apply the changes.
-
-              ```
-              kpt fn render
-              ```
-
-              ```
-              git add . 
-              git commit -m "Add Guardrails solution"
-              git push --set-upstream origin main
-              ```
-
-    c. Cloud Deploy (future)
+      4. Deploy the Config Sync Manifest
+          ```
+          kubectl apply -f root-sync.yaml
+          kubectl wait --for condition=established --timeout=10s crd/rootsyncs.configsync.gke.io
+          ```
+
+      5. Push Configs to Git
+          ```
+          gcloud source repos clone my-lz-repo
+          cd my-lz-repo
+          ```
+
+          Modify the `setters.yaml` file and apply the changes.
+
+          ```
+          kpt fn render
+          ```
+
+          ```
+          git add . 
+          git commit -m "Add Guardrails solution"
+          git push --set-upstream origin main
+          ```
+    ### OCI
+
+      Before we deploy via OCI we have a few things we'll need to do to prepare our environment.
+
+      First we'll need to create an artifact registry to store our OCI artifacts.
+
+      Let's set some environment variables to start
+      ```
+      export PROJECT_ID=PROJECT_ID
+      export AR_REPO_NAME=REPO_NAME
+      export GSA_NAME="$(kubectl get ConfigConnectorContext -n config-control -o jsonpath='{.items[0].spec.googleServiceAccount}' 2> /dev/null)"
+      ```
+
+      Enable Artifact Registry
+      ```
+      gcloud services enable artifactregistry.googleapis.com \
+      --project=${PROJECT_ID}
+      ```
+
+      Create a new repository
+      ```
+      gcloud artifacts repositories create ${AR_REPO_NAME} \
+      --repository-format=docker \
+      --location=northamerica-northeast1 \
+      --description="Config Sync OCI repo" \
+      --project=${PROJECT_ID}
+      ```
+
+      #### Push Config Image to the repository
+
+      ##### Install crane and login to Artifact Registry
+
+      ```
+      go install github.com/google/go-containerregistry/cmd/crane@latest
+      crane auth login northamerica-northeast1-docker.pkg.dev  -u oauth2accesstoken -p "$(gcloud auth print-access-token)"
+      ```
+
+      #### Render the Configs
+
+      At this point we'll want to update our `setters.yaml` with the appropriate values and render the solution configurations using `kpt`.
+
+      ```
+      kpt fn render
```

Comment:
will need to be 
kpt fn render landing-zone

Reply:
Should be fixed now